### PR TITLE
lock down egress by default

### PR DESCRIPTION
**Pull request by phylake**
## Changelog

- lock down ASG egress traffic to allow by default NTP, DNS, HTTP, and HTTPS

## Questions (open the PR then click the check boxes)

Did you update the documentation related to your changes?

- [x] Yes
- [ ] My changes were not already documented

Did you run `make` _before_ committing code and opening this PR?

- [x] Yes
- [ ] I didn't change any code

Did you run `porter create-stack` and `porter sync-stack` to verify provisioning
works?

- [x] Yes
- [ ] N/A

### CODE REVIEW

**Comment by phylake** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/adobe-platform/porter/pull-request/151/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=adobe-platform/porter)

File: provision/map_resource.go [hunk 1]
```diff
@@ -237,19 +233,11 @@ func overwriteASGSecurityGroupEgress(recv *stackCreator, template *cfn.Template,
 		)
 
 		if props, ok = securityGroup["Properties"].(map[string]interface{}); !ok {
```
Comment (davepersing) [inline provision/map_resource.go:235]: constantize `"Properties"` and `"SecurityGroups"`?

**Review by davepersing — COMMENTED** [review]
👍

**Comment by jfieber** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/adobe-platform/porter/pull-request/151/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=adobe-platform/porter)

**Comment by gregoryfischer** [comment]
👍

[Approved with PullApprove](https://pullapprove.com/adobe-platform/porter/pull-request/151/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=adobe-platform/porter)

**Pull request merged**